test/check-storage-home-reuse

```diff
@@ -189,6 +217,19 @@ class TestStorageHomeReuseFedora(VirtInstallMachineCase):
     @skipImage("btrfs support missing on fedora-eln image", "fedora-eln-boot")
     @nondestructive
     def testMultipleRoots(self):
+        """
+        Description:
+            Test the home reuse installation method with a pre-existing multiple Linux OSes
+            on the selected disks
+
+        Expected results:
+            The home reuse installation method is not available
+                - when two Fedora systems are on the selected disks
+                - when no Fedora system is on the selected disks
+
+            The home reuse installation method is available
+                - when only one Fedora system and possibly other Linux OSes are on the selected disks
```

Comment:
This does not reflect the tests (and expected state) correctly.
It should be available only when single Linux OS is available and the OS is Fedora OS.
And it explicitly checks that it is not available in case of
- two Fedora OSes
- single non-Fedora Linux OS is available
- Fedora OS and another Linux OS is available

Reply:
I think I got this now right?